internal/globalgates/globalgates.go

```diff
@@ -20,12 +20,6 @@ var _ = featuregate.GlobalRegistry().MustRegister(StrictlyTypedInputID,
 	featuregate.WithRegisterDescription("Makes type casting rules during configuration unmarshaling stricter. See https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/rfcs/env-vars.md for more details."),
 )
 
-var DisableOpenCensusBridge = featuregate.GlobalRegistry().MustRegister("service.disableOpenCensusBridge",
-	featuregate.StageStable,
-	featuregate.WithRegisterFromVersion("v0.105.0"),
-	featuregate.WithRegisterToVersion("v0.109.0"),
```

Comment:
wait after 0.109.0 is released?

Reply:
The ToVersion represents the release in which the gate is removed:

```
// If the feature stage is either "Deprecated" or "Stable", the "ToVersion" is the Collector release when the feature is removed.
```